### DISCUSSION TITLE
Achieve CII Best Practices Badge -> https://bestpractices.coreinfrastructure.org/en

### DISCUSSION THREAD

**Discussion by ghost** [fastapi/fastapi#6542] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

FastAPI is growing rapidly and I believe as next step for us it make it more available, consumed by large audience and developers, companies to trust & recognise it more. One of the step I think we can take is to join CII Best Practices Badge Program and achieve the best practices recognition.

Below is the link for more info on CII Best Practice Badge Program -
https://bestpractices.coreinfrastructure.org/en

### Wanted Solution

Meet all criteria mentioned below link in phased manner if it is not implemented yet and achieve an official / standard recognition for the project. 

I assume we can sign up and add this project and its you ( @tiangolo ) who can only do it.

Link: https://bestpractices.coreinfrastructure.org/en/criteria/0

### Wanted Code

```python
N/A
```

### Operating System

Other

### FastAPI Version

N/A

### Python Version

N/A

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest, but I'll pass on this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
